syncr_backend/crypto_util.py

```diff
@@ -74,6 +77,39 @@ def generate_private_key() -> rsa.RSAPrivateKey:
     return private_key
 
 
+def sign_dictionary(private_key: rsa.RSAPrivateKey, dictionary: dict) -> bytes:
+    """Takes a dict and returns a rsa signature of the hash of the dict"""
+    signature_interface = private_key.signer(
+        padding.PSS(
+            mgf=padding.MGF1(hashes.SHA256()),
+            salt_length=padding.PSS.MAX_LENGTH,
+        ),
+        hashes.SHA256(),
+    )
+
+    signature_interface.update(hash(bencode.encode(dictionary)))
+    return signature_interface.finalize()
+
+
+def verify_signed_dictionary(
+    public_key: rsa.RSAPublicKey,
+    signature: bytes,
+    dictionary: dict,
+) -> None:
+    '''Returns None if success,
+     else throws cryptography.exceptions.InvalidSignature'''
```

Comment:
Comment in proper format (also function descriptions should be """tripple double quotes""")